lib/runctl.js

```diff
@@ -155,6 +172,29 @@ function onRequest(req, callback) {
   }
 }
 
+function requestAllTargets(req, callback) {
```

Comment:
Correct, the set/unset is applied live to everything. That was the point, I thought. Do we want `runctl` to print a warning? I'd rather not, but if we do, maybe something like "Not all modules take advantage of dynamic environments and your app may need a soft-restart for the changes to take full effect."